static/src/javascripts/bootstraps/commercial.js

```diff
@@ -40,7 +40,7 @@ const commercialModules: Array<Array<any>> = [
     ['cm-closeDisabledSlots', closeDisabledSlots.init],
     ['cm-stickyTopBanner', stickyTopBanner.init],
     ['cm-fill-advert-slots', fillAdvertSlots, true],
-    ['cm-paidContainers', paidContainers.init],
+    ['cm-paidContainers', paidContainers()],
```

Comment:
You just need the function here, without invoking it (Intellij auto-completes it like this all the time 😫 )